### PR TITLE
User progress

### PR DESCRIPTION
**Pull request by npnote8**
get max Score, last Score and last attempt number for a quiz by user
userProgress postman collection is attached

Naming:
{
quiz: quizId,
maxScore: bestScore,
score: lastScore,
attemptNumber: attemptsCount,
user: userId}

### CODE REVIEW

File: src/models/attemptModel.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const mongoose = require("mongoose");
+const attemptSchema = new mongoose.Schema({
+  test: {
```
Comment (ezbc) [inline src/models/attemptModel.js]: I'm inclined to rename `test` to `quiz` to keep the language consistent. What do you think?

File: src/controllers/attemptController.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    ]);
+    return attempts;
+    //
+  } catch (error) {
+    return error;
+  }
+};
+
+// get latest score for every quiz by user: sortBy attempNumber in descending order
+const latestScoreAttempts = async (userId) => {
```
Comment (ezbc) [inline src/controllers/attemptController.js]: I found it a little difficult to understand which function in this file was the controller. I generally try to keep only controller functions in controller files and import any extra functions needed for the controller. What do you think of moving latestScoreAttempts and maxScoreAttempts to another file? Perhaps src/attempts.js?

File: Quizz App.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,365 @@
...
+								"admin"
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		}
+	]
+}
```
Comment (ezbc) [inline Quizz App.postman_collection.json:365]: Not a big deal, but I didn't see any postman requests for the attempts. Did you intend to add any?

**Review by ezbc — CHANGES_REQUESTED** [review]
Great work Alina! I have only small comments for you to look into then lets get this merged!

File: src/models/attemptModel.js (hunk 1 above)
Comment (npnote8) [reply src/models/attemptModel.js]: Renamed test to quiz to keep the language consistent.

File: src/controllers/attemptController.js (hunk 2 above)
Comment (npnote8) [reply src/controllers/attemptController.js]: Moved latestScoreAttempts and maxScoreAttempts to src/functions/attempts.js

File: Quizz App.postman_collection.json (hunk 3 above)
Comment (npnote8) [reply Quizz App.postman_collection.json:365]: attempts postman collection is in userProgress.postman_collection.json

**Review by ezbc — APPROVED** [review]
Since time is short and there's no deal-breakers, lets merge

**Review by ezbc — APPROVED** [review]

**Review by ezbc — APPROVED** [review]

**Pull request merged**